schema/swa-cli.config.schema.json

```diff
@@ -0,0 +1,100 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "properties": {
+    "configurations": {
+      "additionalProperties": {
+        "allOf": [
+          {
+            "properties": {
+              "apiLocation": {
+                "description": "API folder or Azure Functions emulator address",
+                "type": "string"
+              },
+              "apiPort": {
+                "description": "API backend port",
+                "type": "number"
+              },
+              "apiPrefix": {
+                "enum": ["api"],
+                "type": "string"
+              },
+              "appArtifactLocation": {
```

Comment:
Question: Is there an option to mark `appArtifactLocation` as deprecated? In SWA, this argument has been renamed to `outputLocation`.

Reply:
If we mark it as deprecated would you want to include an option for `outputLocation`?

Reply:
And the best way I found for marking a property as deprecated would be to just include it in the property description.

Reply:
That'd be enough I think.